currency/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Simple Currency System
+Help Site: [This is Optional]/<> <-This is Not Optional
+
+There Will be More Commands In the Future :)
+
+Simple Currency System with Changeable Name/Emoji Money.
+
+It Works With Some Of The Fun Commands that work with Currency, ex: slotsMachine.cc.lua or guessNumber.cc.lua
+
+These Commands Are Standalone, You can add Only one, all or two
+
+# Some Current Commands It Have:
+1. **-balance [User/Member]**
+   It Gets The Current Balance of A User Member or if no arguments given, The Users Balance, ex: !balance
+1. **-addbalance <User/Member> <Amount>** 
+   It adds Balance To A User/Member, ex: !addbalance @Gas_Infinity 100
+1. **-rob <User/Member>** 
+   It robs a Random Amount from a minAmount to the max Money The Player Have, It Detects if you or The Player Have The Min Amount and If you`re Robbing to Yourself, Also it Have A Very Simple         Cooldown System. ex: !rob @Gas_Infinity
+1. **-work** 
+   Get A Random Amount with Configurable MinAmount or MaxAmount Given, Configurable WinMsg And LoseMsg. ex: -work
+1. **-deposit < all > or < amount >** 
+   Deposit Money To The Bank, You can change the minamount required for Deposit. ex: -deposit 10
+1. **-withdraw < all > or < amount >** 
+   Withdraw Money from The Bank, You can change the minamount required for withdraw. ex: -withdraw 10
+1. **-money-leaderboard < Module > [Page]** 
+   Check The Money Leaderboards. The Modules are: In Hand(-h), In The Bank(-b) and Networth [Bank+Hand] (-n). ex: -money-leaderboards -n
+1. **"Birthday Update":**
+   BirthDay Update For Giving An Amount of Money in The Birthday Of A Person, You Can Activate it Or Deactivate It (In BirthDay Folder)

```

Comment:
```suggestion
1. **-work** 
   Get A Random Amount with Configurable MinAmount or MaxAmount Given, Configurable WinMsg And LoseMsg. ex: -work
2. **-deposit < all > or < amount >** 
   Deposit Money To The Bank, You can change the minamount required for Deposit. ex: -deposit 10
3. **-withdraw < all > or < amount >** 
   Withdraw Money from The Bank, You can change the minamount required for withdraw. ex: -withdraw 10
4. **-money-leaderboard < Module > [Page]** 
   Check The Money Leaderboards. The Modules are: In Hand(-h), In The Bank(-b) and Networth [Bank+Hand] (-n). ex: -money-leaderboards -n
5. **"Birthday Update":**
   BirthDay Update For Giving An Amount of Money in The Birthday Of A Person, You Can Activate it Or Deactivate It (In BirthDay Folder)
```
Same as first review comment, use incrementing numbers for ordered lists.

Reply:
Changed